man/org.freedesktop.systemd1.xml

```diff
@@ -2605,6 +2605,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly t LimitRTTIMESoft = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s ExecSearchPaths = '...';
```

Comment:
oh, I stil lthink we should store and expose this in normalized form, i.e. as array of strings instead of a concatenated string. Using the colon notation in textual output, and in env vars is fine, but internally this is an array of strings, and should be stored and exposed via IPC like that.

i.e. we should parse data when we first come into contact with it, then keep it around in normalized, parsed form. Parsing late when we use it is too late.

Reply:
I'd name this in singular form `ExecSearchPath` really, even if it takes multiple of them. I think it would be surprising using plural here, because `$PATH` is singular too